crates/bones_render/src/camera.rs

```diff
@@ -1,6 +1,6 @@
 //! Camera components.
 
-use crate::prelude::*;
+use crate::{color::Color, prelude::*};
```

Comment:
Could you add `Color` to the prelude.

And here, too:

https://github.com/fishfolk/bones/pull/110/files#diff-a755dd4667b7d1e7c74a892f3396c353b2945aef60faa131c460c197b764564eR3-R4

Reply:
silly me. that should have been there! nice catch